server/container_create.go

```diff
@@ -476,50 +476,53 @@ func (s *Server) CreateContainer(ctx context.Context, req *pb.CreateContainerReq
 		return nil, errors.Wrap(err, "setting container name and ID")
 	}
 
+	cleanupFuncs := make([]func(), 0)
```

Comment:
I think `, 0` is redundant

Reply:
you need to specify a length when initializing a slice

Reply:
Right, I always forget about this.

```go
cleanupFuncs := []func(){}
```
maybe?

Reply:
is there any reason that is better than using make (which feels more idiomatic to me)